packages/product-feed-zbozi/src/Model/FeedItem/ZboziFeedItem.php

```diff
@@ -0,0 +1,248 @@
+
+    /**
+     * @var string[]
+     */
+    protected $parametersByName;
+
+    /**
+     * @var float|null
+     */
+    protected $cpc;
+
+    /**
+     * @var float|null
+     */
+    protected $cpcSearch;
+
+    public function __construct(
+        int $id,
+        ?int $mainVariantId,
+        string $name,
+        ?string $description,
+        string $url,
+        ?string $imgUrl,
+        ?string $brandName,
+        ?string $ean,
+        ?string $partno,
+        ?int $availabilityDispatchTime,
+        Price $price,
+        array $pathToMainCategory,
+        array $parametersByName,
+        ?float $cpc,
+        ?float $cpcSearch
+    ) {
+        $this->id = $id;
+        $this->mainVariantId = $mainVariantId;
+        $this->name = $name;
+        $this->description = $description;
+        $this->url = $url;
+        $this->imgUrl = $imgUrl;
+        $this->brandName = $brandName;
+        $this->ean = $ean;
+        $this->partno = $partno;
+        $this->availabilityDispatchTime = $availabilityDispatchTime;
+        $this->price = $price;
+        $this->pathToMainCategory = $pathToMainCategory;
+        $this->parametersByName = $parametersByName;
+        $this->cpc = $cpc;
+        $this->cpcSearch = $cpcSearch;
+    }
+
+    /**
+     * @return int
+     */
+    public function getSeekId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * @return int
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * @return int|null
+     */
+    public function getGroupId(): ?int
+    {
+        return $this->mainVariantId;
+    }
+
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getDescription(): ?string
+    {
+        return $this->description;
+    }
+
+    /**
+     * @return string
+     */
+    public function getUrl(): string
+    {
+        return $this->url;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getImgUrl(): ?string
+    {
+        return $this->imgUrl;
+    }
+
+    /**
+     * @return \Shopsys\FrameworkBundle\Model\Pricing\Price
+     */
+    public function getPrice(): Price
+    {
+        return $this->price;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getEan(): ?string
+    {
+        return $this->ean;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getProductno(): ?string
+    {
+        return $this->partno;
+    }
+
+    /**
+     * @return int|null
+     */
+    public function getDeliveryDate(): ?int
+    {
+        return $this->availabilityDispatchTime;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getManufacturer(): ?string
+    {
+        return $this->brandName;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getCategoryText(): ?string
```

Comment:
based on the naming conventions, this should be probably named `getCategorytext` but please do not do that :laughing: